_data/wallets.yml

```diff
@@ -86,6 +86,17 @@ websites:
       btc: Yes
       othercrypto: No
 
+    - name: Edge
+      url: https://edgesecure.co
+      img: edge.png
+      twitter: EdgeSecure
+      facebook: @EdgeSec
```

Comment:
Gotta catch these. "2" in the facebook name shouldnt be able to pass. Shall figure out a better way of relaying this from our travis stuff